database/seeds/MinecraftServiceTableSeeder.php

```diff
@@ -243,6 +243,41 @@ private function addCoreOptions()
             'startup' => null,
             'script_install' => $script,
         ]);
+
+        $script = <<<'EOF'
+#!/bin/ash
+# Forge Installation Script
+#
+# Server Files: /mnt/server
+apk update
+apk add curl openjdk8
```

Comment:
Why not use the existing java docker containers that are out there and likely already on the user's system?